packages/calypso-e2e/src/lib/pages/signup/signup-pick-plan-page.ts

```diff
@@ -28,10 +33,7 @@ export class SignupPickPlanPage {
 	 * @returns {Promise<SiteDetails>} Details of the newly created site.
 	 */
 	async selectPlan( name: Plans ): Promise< NewSiteResponse > {
-		await Promise.all( [
-			this.page.waitForURL( /.*start\/plans.*/ ),
-			this.page.waitForLoadState(),
-		] );
+		await Promise.all( [ this.page.waitForURL( plansPageUrl ), this.page.waitForLoadState() ] );
```

Comment:
No need to wait for the load state explicitly since [it's internally done](https://playwright.dev/docs/api/class-frame#frame-wait-for-url-option-wait-until) by the `waitForURL` method.